lightning/src/ln/chanmon_update_fail_tests.rs

```diff
@@ -165,11 +165,12 @@ fn do_test_simple_monitor_temporary_update_fail(disconnect: bool) {
 	let events_3 = nodes[1].node.get_and_clear_pending_events();
 	assert_eq!(events_3.len(), 1);
 	match events_3[0] {
-		Event::PaymentClaimable { ref payment_hash, ref purpose, amount_msat, receiver_node_id, via_channel_id, .. } => {
+		Event::PaymentClaimable { ref payment_hash, ref purpose, amount_msat, receiver_node_id, ref via_channel_id_pairs, .. } => {
 			assert_eq!(payment_hash_1, *payment_hash);
 			assert_eq!(amount_msat, 1_000_000);
 			assert_eq!(receiver_node_id.unwrap(), nodes[1].node.get_our_node_id());
-			assert_eq!(via_channel_id, Some(channel_id));
+			assert_eq!(via_channel_id_pairs.len(), 1);
+			assert_eq!(via_channel_id_pairs.last().unwrap().0, channel_id);
```

Comment:
Can't this simply be `assert_eq!(via_channel_id_pairs, vec![(channel_id, other_value)]);` ?

Reply:
That makes sense! Also this help us expand the test suite, covering for user_channel_id, also with channel_id
Updated in [pr3655.12](https://github.com/shaavan/rust-lightning/commits/pr3655.12). Thanks for the pointer!